splash-lp-indexer/src/config.rs

```diff
@@ -23,6 +23,7 @@ pub struct AppConfig {
     pub harvest_limits: HarvestLimits,
     pub splash_policy_id_hex: String,
     pub ve_config: VeConfig,
+    pub genesis_epoch_start_time: u64,
```

Comment:
This seems to duplicate epoch_start: Slot in ve_config? If not, then these two numbers must be coherent, can we derive one from another or do some consistency check at least?

Reply:
It's a duplicate. Removed it.